test/test_helper.rb

```diff
@@ -18,4 +18,12 @@ class ActiveSupport::TestCase
   fixtures :all
 
   # Add more helper methods to be used by all tests here...
+  def check_response(expected_type:, expected_status: :success)
```

Comment:
What do you think about putting the check_response method in test_helper so it can be accessed by all the tests?

Reply:
Yes I approve of this and it would make things cleaner